src/modules/partition/PartitionViewStep.cpp

```diff
@@ -698,6 +698,8 @@ PartitionViewStep::setConfigurationMap( const QVariantMap& configurationMap )
                 Calamares::getBool( configurationMap, "alwaysShowPartitionLabels", true ) );
     gs->insert( "enableLuksAutomatedPartitioning",
                 Calamares::getBool( configurationMap, "enableLuksAutomatedPartitioning", true ) );
+    gs->insert( "preCheckEncryption",
+                Calamares::getBool( configurationMap, "preCheckEncryption", false ) );
```

Comment:
The ChoicePage has access to the config.  I don't think we need to use global storage here.